object/fmt.go

```diff
@@ -105,10 +118,14 @@ func CalculateAndSetSignature(key *ecdsa.PrivateKey, obj *Object) error {
 	return nil
 }
 
+// VerifyIDSignature verifies object ID signature.
 func VerifyIDSignature(obj *Object) error {
```

Comment:
Should be the method (lets return `bool` instead of `error`).

Reply:
> Should be the method

still do not get it

> lets return bool instead of error

agree

Reply:
fixed